src/agentlab/experiments/graph_execution_ray.py

```diff
@@ -0,0 +1,39 @@
+# import os
+
+# # Disable Ray log deduplication
+# os.environ["RAY_DEDUP_LOGS"] = "0"
+
+import ray
+import bgym
+from agentlab.experiments.exp_utils import run_exp
+
+
+run_exp = ray.remote(run_exp)
+
+
+def execute_task_graph(exp_args_list: list[bgym.ExpArgs], avg_step_timeout=30):
```

Comment:
I get it but I was a bit confused with the `task` naming which is used for browsergym tasks in other parts of the code. Maybe change `task` for `exp` here ?